### ISSUE TITLE
Set team descriptions

### ISSUE THREAD

**Issue by bobheadxi** [ubclaunchpad/rocket#70]

**Comment by bfbachmann** [comment]
This would be a great feature.

Along with this we'll need to update the website to display these descriptions. I'm thinking we should probably just redesign the `Team` section, @bobheadxi. What do you think?

**Comment by bobheadxi** [comment]
Yep I think the Team section is due for a bit of a redesign - I'll look into it this weekend, and implement it once this ticket gets closed

Relevant ticket: https://github.com/ubclaunchpad/ubclaunchpad.github.io/issues/52

Also added that a setting both a photo and a description would be nice too, just for potentially more visual content
